docs/CODING_STYLE.md

```diff
@@ -69,6 +69,14 @@ layout: default
   ```
 
 - Do not write `foo ()`, write `foo()`.
+- `else` blocks should generally start on the same line as the closing `}`:
+  ```c
+  if(foobar) {
```

Comment:
there should be a space between `if` and `(foobar)`

Reply:
Thanks, that was just sloppy of me. Fixed now.